lib/assets/javascripts/cartodb/table/overlays/overlay_properties_bar.js

```diff
@@ -198,7 +198,13 @@ cdb.admin.OverlayPropertiesBar =  cdb.core.View.extend({
   render: function() {
 
     this._addForm();
-    this._addActions();
+
+    // disable 'Actions' edit section for fixed title/description added via map options
+    // (both report their type as 'header')
+    var modelType = this.model.attributes.type;
+    if (modelType != 'header') {
```

Comment:
So there is, nice find. The method call happens from a generalized bound event, so I went with modifying the default setter in the method.